core/controllers/project.py

```diff
@@ -28,109 +26,12 @@ def get_project(project_id: str):
     :param project_id: the project_id provided as a string
     :return: all data associated with a project
     """
-    project_data = get_by_project_id(table="project", project_id=project_id)
+    project = Project.query.filter_by(project_id=project_id).first()
 
-    if not project_data:
+    if not project:
```

Comment:
you're right, this is now "disgustingly simple" 🙌

Reply:
😆